src/main/java/sg/ncl/testbed_interface/User2.java

```diff
@@ -1,7 +1,12 @@
 package sg.ncl.testbed_interface;
 
+import sg.ncl.validation.Validator;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'sg.ncl.validation.Validator'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUselessImportCheck)